src/Components/DetailsDisplay/DetailsDisplay.js

```diff
@@ -24,24 +25,35 @@ const DetailsDisplay = ({ movieToDisplay, trailersToDisplay }) => {
         <p className='details__movie-rating'>
           {`Tater Rating: ${rating}`}
         </p>
-        <p className='details__movie-release'>
-          {`${releaseYear} | ${movieToDisplay.runtime} mins`}
-        </p>
-        <p className='details__movie-overview'>
-          {movieToDisplay.overview}
-        </p>
-          {!genres && 
-          <div> No Genres to display
-          </div>}
-          {genres && 
-          <div className='details__movie-genres'>
-          Genre: {genres}
-          </div>}
+
+        {releaseYear && runtime !== 0 && <p className='details__movie-release'>
+          {`${releaseYear} | ${runtime} mins`}
+        </p>}
+        {releaseYear && runtime === 0 && <p className='details__movie-release'>
+          {`${releaseYear}`}
+        </p>}
+        {!releaseYear && runtime !== 0 && <p className='details__movie-release'>
+          {`${runtime} mins`}
+        </p>}
+        
+        {movieToDisplay.overview && <p className='details__movie-overview'> {movieToDisplay.overview} </p>}
+        {!movieToDisplay.overview && <p>No synopsis provided</p>}
```

Comment:
Appreciate how you put the conditional rendering together for this - glad you noticed the sparse info provided for that movie!